index.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   included(app) {
     this._super.included.apply(this, arguments);
     this.import('vendor/shims/esri-loader.js');
-    this.options = Object.assign({ additionalFiles: [] }, app.options[this.moduleName()])
+    this.addonOptions = Object.assign({ additionalFiles: [] }, app.options[this.moduleName()])
```

Comment:
This addresses a build-time deprecation saying

```
DEPRECATION: Ember CLI addons manage their own module transpilation during the `treeForAddon` 
processing. `ember-esri-loader` (found at `...`) has overridden the `this.options.babel` options 
which conflicts with the addons ability to transpile its `addon/` files properly. Falling back to 
default babel configuration options.
```

Looks like I introduced this one with https://github.com/Esri/ember-esri-loader/pull/115 by overwriting `this.options`.